schema-registry/rest-service/src/test/resources/schema-registry-client.yaml

```diff
@@ -8,4 +8,11 @@ schema.registry.client.schema.metadata.cache.expiry.interval : 300
 schema.registry.client.schema.text.cache.size : 1024
 schema.registry.client.schema.text.cache.expiry.interval : 300
 schema.registry.client.url.selector : "com.hortonworks.registries.schemaregistry.client.FailoverUrlSelector"
-jersey.config.client.readTimeout: 300000
\ No newline at end of file
+jersey.config.client.readTimeout: 300000
+schema.registry.client.retry.strategy:
+  className: com.hortonworks.registries.schemaregistry.retry.policy.ExponentialBackoffRetryPolicy
+  config:
+    sleepTimeMs: 100
+    maxRetries: 3
+    maxSleepTimeMs: 3000
+    exponent: 3
```

Comment:
The exponential backoff retry with exponent hardcoded as 2 and configurable `sleepTimeMs` would be sufficient in most of the cases. (eg) With sleepTimeMs as 1000. The retry would happen after 1000, 2000, 4000, 8000, 16000, 32000, 64000... Do you have any reason why user should configure the `exponent`?

Reply:
In most of the cases, exponent 2 should suffice. As a result, ExponentialbackoffRetryPolicy has 2 as default. I don't see any harm in exposing this, as it will let users further fine-tune their retry mechanism.